src/ngx_lua_resty_lmdb_module.c

```diff
@@ -111,6 +171,9 @@ static ngx_int_t ngx_lua_resty_lmdb_init_worker(ngx_cycle_t *cycle)
     ngx_lua_resty_lmdb_conf_t *lcf;
     int                        rc;
     int                        dead;
+    MDB_val                    enckey;
+    int                        block_size = 0;
+    u_char                     keybuf[2048];
```

Comment:
Is it OK? because this buffer is local storage.

Reply:
Is it about security concern or data race concern?

Reply:
I am not familiar with openssl, I am not sure about this usage, it looks OK.